schema/tables/orbit_info.yml

```diff
@@ -33,5 +33,9 @@ columns:
     type: text
     required: false
     description: The Update Framework update channel used for the Fleet Desktop executable.
+  - name: uptime
+    type: text
+    required: false
+    description: Uptime of the orbit process.
```

Comment:
```suggestion
    description: Uptime of the orbit process in seconds.
```

Reply:
It's actually a Go duration string (e.g. `1m52s`) meant only for troubleshooting. Is it more useful if it's seconds?

Reply:
I see, I didn't realize type was text and not number. Yes, config string seems fine, although not consistent with uptime columns in other tables.

Reply:
I like consistency (plus it will be easier to process in the future):
[Use seconds for uptime for consistency and easier processing](https://github.com/fleetdm/fleet/pull/15848/commits/e0af7a2d3851df0b9499fcd14b35eb017a412639)